hamocc/dipowa.F90

```diff
@@ -208,9 +208,9 @@ subroutine dipowa(kpie,kpje,kpke,omask)
                 &  ( sedb1(i,l,iv) - tredsy(i,l,3) * powtra(i,j,l+1,iv) )      &
                 &  / tredsy(i,l,2)
 
-           ! used in inventory_bgc/maschk (diagnostics)
+           ! diffusive fluxes (positive downward)
            sedfluxo(i,j,iv) = sedfluxo(i,j,iv)                                 &
-                &  + ocetra(i,j,kbo(i,j),iv) - aprior
+                &  -(ocetra(i,j,kbo(i,j),iv) - aprior)* bolay(i,j)
```

Comment:
I think this is correct as it is: In `powach` `sedfluxo` is set to zero when entering the sediment. Then it is receiving a value for silicate and oxygen (but not the others) in `powach`. After that `dipowa` is called, where sedfluxo is updated in a loop. In order to avoid overwriting the values for oxygen and silicate, it is written as above.